### ISSUE TITLE
Pre-trained NN examples

### ISSUE THREAD

**Issue by romansavrulin** [hamuchiwa/AutoRCCar#51]
Hello! 

I'm working on porting this project completely to Raspberry pi. Can you provide pre-trained NN examples to allow me quickly evaluate performance on the Pi board?  This will save me a lot of time because I have to use another approach to control RC car and getting training data for NN.

**Comment by hamuchiwa** [comment]
Hi, I think a quick way to test your components is to generate random data with labels, and then do the training. It's more flexible in this way in case of different image size or parameters.

**Comment by AiBuddy** [comment]
@hamuchiwa  how to generate random data with labels?
